pkg/manager-server/file_system_gfs2.go

```diff
@@ -104,6 +105,27 @@ func (f *FileSystemGfs2) Create(devices []string, opts FileSystemOptions) error
 		return err
 	}
 
+	userID := 0
+	if _, exists := opts[UserID]; exists {
+		userID = opts[UserID].(int)
+	}
+
+	groupID := 0
+	if _, exists := opts[GroupID]; exists {
+		groupID = opts[GroupID].(int)
+	}
+
+	mountpath := "/mnt/nnf/client/" + f.name
+	if err := f.Mount(mountpath); err != nil {
```

Comment:
So if Create does the mount, what does the call to Mount() do? I don't like that the full mountpath is no longer decided by the nnf-ec client; seems like a bad dependency.

What you could do is cache the userID/groupID in the resource on create, then call Chown() in the mount call.

Reply:
Follow-on; if the rabbit crashes, nnf-ec must do the Chown() when it's reconstructing the state; so I think the UserID/GroupID need to be persisted as metadata

Reply:
The Chown() has to happen at Create() time so that it only happens once. If it happened during the Mount(), then any group/user ID changes a user had made to the base directory would be overwritten. That wouldn't be correct behavior for persistent instances.

The Mount() that's happening here is only to run the Chown(), and then it's immediately unmounted. The mount path for mounts coming from an NnfAccess for data in/out still use the same paths they were using before.

Reply:
Ohh I totally missed that. Thanks.

Reply:
Yep. I can add a comment.